integration-tests/validation/src/androidTest/java/com/android/designcompose/testapp/validation/InteractionTests.kt

```diff
@@ -45,36 +44,36 @@ class InteractionTests {
 
         // Navigate to the Triggers -> Timeouts menu
         composeTestRule.onNodeWithText("Triggers").performClick()
-        composeTestRule.onNodeWithText("While Pressed").assertIsDisplayed().performClick()
-        composeTestRule.onNodeWithText("Timeouts").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithText("While Pressed").assertExists().performClick()
+        composeTestRule.onNodeWithText("Timeouts").assertExists().performClick()
```

Comment:
Forgot to ask -- why was `assertIsDisplayed` replaced with `assertExists`?

Reply:
I was trying to figure out why the Tutorial app test always failed on Gradle Managed Device tests (like `./gradlew tabletAtdApi30Check`). I realized that the Gradle managed devices always start up in landscape mode, and for some reason Compose doesn't think the main screen of the Tutorial app is visible in landscape (it may have something to do with the main screen being taller than the entire screen when the device is in landscape). Switching things to "assertExists()" fixes it.